content/it/distributed-apps.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Applicazione distribuita
+status: Feedback Appreciated
+category: concetto
+---
+
+## Cos'è
+
+Un'applicazione distribuita è un'applicazione la cui funzionalità è suddivisa in parti più piccole e indipendenti. Le applicazioni distribuite sono generalmente composte da [microservizi](/it/microservices/) che gestiscono diversi aspetti all'interno dell'applicazione più ampia. In un ambiente cloud native i singoli componenti sono eseguiti come container all'interno di un [cluster](/it/cluster/).
+
+## Quali problematiche affronta
+
+Un'applicazione in esecuzione su un singolo computer rappresenta un singolo punto di vulnerabilità: se quel computer si blocca, l'applicazione non sarà più disponibile. Le applicazioni distribuite vengono spesso messe in contrapposizione alle applicazioni monolitiche. Un'applicazione monolitica può essere più difficile da scalare perché i singoli componenti non possono essere scalati in modo indipendente. Mentre cresce può anche diventare un ostacolo alla developer velocity, perché più sviluppatori devono lavorare su una codebase condivisa che non ha necessariamente dei confini ben definiti.
```

Comment:
quella di 'scalare' è una annosa questione, in cui sostengo che usare il verbo come traduzion di "to scale" sia in realtà improprio. 
Ho provato a trovare una locuzione anche per la developer velocity, ma non ho trovato nulla di soddsfacente. @annalisag-spark qualche idea a riguardo?
```suggestion
Un'applicazione in esecuzione su un singolo computer rappresenta un singolo punto di vulnerabilità: se quel computer si blocca, l'applicazione non sarà più disponibile. Le applicazioni distribuite vengono spesso messe in contrapposizione alle applicazioni monolitiche. Un'applicazione monolitica è più difficilmente scalabile perché i singoli componenti non sono scalabili indipendentemente. Man mano che questi ultimi crescono, possono diventare anche un ostacolo alla developer velocity, perché più sviluppatori devono lavorare su una codebase condivisa che non ha necessariamente dei confini ben definiti.
```

Reply:
Ok per "to scale": secondo me ormai può essere accettato anche scalare vista la sua larghissima diffusione, ma l'importante è scegliere una linea ed essere coerenti nelle varie pagine.

Developer Velocity è ormai un concetto assestato e conosciuto secondo me, quindi non lo tradurrei. 
Nel caso possiamo proporre di aggiungere questo termine al glossario generale e poi tradurlo, in modo che anche chi non dovesse essere familiare può capire di cosa si parla. In ogni caso, però, lo terrei come step successivo.